src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java

```diff
@@ -9,18 +9,13 @@
 import in.projecteka.consentmanager.common.CentralRegistryTokenVerifier;
 import in.projecteka.consentmanager.common.IdentityService;
 import in.projecteka.consentmanager.link.ClientErrorExceptionHandler;
+import in.projecteka.consentmanager.user.LockedUsersRepository;
 import in.projecteka.consentmanager.user.TokenService;
+import in.projecteka.consentmanager.user.UserRepository;
 import io.vertx.pgclient.PgConnectOptions;
 import io.vertx.pgclient.PgPool;
 import io.vertx.sqlclient.PoolOptions;
-import org.springframework.amqp.core.AmqpAdmin;
-import org.springframework.amqp.core.Binding;
-import org.springframework.amqp.core.BindingBuilder;
-import org.springframework.amqp.core.DirectExchange;
-import org.springframework.amqp.core.Exchange;
-import org.springframework.amqp.core.ExchangeBuilder;
-import org.springframework.amqp.core.Queue;
-import org.springframework.amqp.core.QueueBuilder;
+import org.springframework.amqp.core.*;
```

Comment:
Don't use wildcard imports.

Reply:
done

Reply:
Don't resolve it by yourselves, reviewer will resolve it.

Reply:
Ok, cool

Reply:
It still has the wildcard...change the settings of intellij -> preferences -> editor -> Code style -> java -> imports -> Class count to use import with '*' - 999

Reply:
done